triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -399,37 +397,6 @@ abstract class AbstractTriggerServiceTest extends AsyncFlatSpec with Eventually
       } yield succeed
   }
 
-  it should "give an 'unauthorized' response for a stop request without an authorization header" in withTriggerService(
-    None) { (uri: Uri, _, _) =>
-    val uuid: String = "ffffffff-ffff-ffff-ffff-ffffffffffff"
-    val req = HttpRequest(
-      method = HttpMethods.DELETE,
-      uri = uri.withPath(Uri.Path(s"/v1/stop/$uuid")),
-    )
-    for {
-      resp <- Http().singleRequest(req)
-      _ <- resp.status should equal(StatusCodes.Unauthorized)
-      body <- responseBodyToString(resp)
-      JsObject(fields) = body.parseJson
-      _ <- fields.get("status") should equal(Some(JsNumber(StatusCodes.Unauthorized.intValue)))
-      _ <- fields.get("errors") should equal(
-        Some(JsArray(JsString("missing Authorization header with Basic Token"))))
-    } yield succeed
-  }
-
-  it should "give an 'unauthorized' response for a start request with an invalid party identifier" in withTriggerService(
```

Comment:
In principle, we could continue validating party formats in the trigger service. However, this feels like something that is better left to the ledger so I’ve removed it.